smart-events-user-dashboard/src/index.css

```diff
@@ -26,10 +26,10 @@ a:hover {
 
 body {
   margin: 0;
-  display: flex;
-  place-items: center;
+  /*display: flex;
+  place-items: center;*/
   min-width: 320px;
-  min-height: 100vh;
+  height: 100vh;
```

Comment:
Incredible. This is the greatest trick I ever learned using CSS. Great work finding and using it.